packages/react-tooltip/src/components/Tooltip/useTooltip.tsx

```diff
@@ -35,50 +27,60 @@ const arrowHeight = 6; // Update the arrow's width/height in useTooltipStyles.ts
  * @param ref - reference to root HTMLElement of Tooltip
  * @param defaultProps - (optional) default prop values provided by the implementing type
  */
-export const useTooltip = (
-  props: TooltipProps,
-  ref: React.Ref<HTMLElement>,
-  defaultProps?: TooltipProps,
-): TooltipState => {
+export const useTooltip = (props: TooltipProps, ref: React.Ref<HTMLDivElement>): TooltipState => {
   const context = React.useContext(TooltipContext);
   const isServerSideRender = useIsSSR();
   const { targetDocument } = useFluent();
   const [setDelayTimeout, clearDelayTimeout] = useTimeout();
 
-  const state = mergeProps(
-    {
-      ref,
-      children: props.children,
-      content: {
-        as: React.Fragment,
-      },
-      id: useId('tooltip-'),
-      role: 'tooltip',
-      showDelay: 250,
-      hideDelay: 250,
-      triggerAriaAttribute: 'label',
-    },
-    defaultProps && resolveShorthandProps(defaultProps, tooltipShorthandProps),
-    resolveShorthandProps(props, tooltipShorthandProps),
-  );
-
-  const [visible, setVisibleInternal] = useControllableState({ state: state.visible, initialState: false });
+  const {
+    content,
+    inverted,
+    withArrow: pointing,
```

Comment:
This looks a bit strange, can you please clarify why rename to `pointing` is required?

Reply:
This change came from `master` and I screwed it up on merging, so I kind of re-implemented

Reply:
Let's remove it if it is not breaking anything